### PR TITLE
VRS: Dropout augmentation for additional selected features

### PR DESCRIPTION
**Pull request by torbjorgen**

### CODE REVIEW

**Comment by torbjorgen** [comment]
Waiting for job 8565609, /home/tor.bjorgen/repos/cg/binarycross-w-augmentation

**Comment by torbjorgen** [comment]
slurm-8565609.out was evicted before training outperformed current baseline.
Blue is training performance, others are baseline.
# Train
However, the trend looks really promising on the training set data.
image


# Test
On the test data however, it's seems like the model is overfitting
and performance is poorer than existing baseline?
Overfitting is contradicted by the fact that test loss is still decreasing,
but the TPs are decreasing. Is this a data or model misbehavior? 
Red is this PR model, others are baseline.
image

**Comment by torbjorgen** [comment]
Difference in behavior most likely due to model learning dropout behavior in training data, which is not present in validation data.
Hence, model is overfitting to dropout behavior and failing on validation data.

Actions: decrease dropout probability or remove features from dropout.
